sprint1/style.css

```
@@ -0,0 +1,531 @@
+@font-face {
+    font-family: 'Pretendard';
+    src: url('https://cdn.jsdelivr.net/gh/Project-Noonnu/noonfonts_2107@1.1/Pretendard-Regular.woff') format('woff');
+    font-weight: 400;
+    font-display: swap;
+  }
+  @font-face {
+    font-family: 'Pretendard';
+    src: url('https://cdn.jsdelivr.net/gh/Project-Noonnu/noonfonts_2107@1.1/Pretendard-Medium.woff') format('woff');
+    font-weight: 500;
+    font-display: swap;
+  }
+  @font-face {
+    font-family: 'Pretendard';
+    src: url('https://cdn.jsdelivr.net/gh/Project-Noonnu/noonfonts_2107@1.1/Pretendard-SemiBold.woff') format('woff');
+    font-weight: 600;
+    font-display: swap;
+  }
+  @font-face {
+    font-family: 'Pretendard';
+    src: url('https://cdn.jsdelivr.net/gh/Project-Noonnu/noonfonts_2107@1.1/Pretendard-Bold.woff') format('woff');
+    font-weight: 700;
+    font-display: swap;
+  }
+*{
+
+      
+    box-sizing: border-box;
+}
+
+a{
+    text-decoration: none;
+}
+
+body{
+    margin: 0;
+    padding: 0;
+    font-family: 'Pretendard';
+}
+
+header{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 9.5px 200px;
+    background-color: #ffffff;
+
+}
+@media (min-width: 744px) and (max-width: 1120px) {
+    header{
+        padding: 9.5px 20px;
+    }
+}
+
+@media (min-width: 375px) and (max-width: 744px) {
+    header{
+        padding: 9.5px 20px;
+    }

```

💊 제안
이렇게 미디어쿼리 조건에 min을 추가해주시기보다 상속받게 해주시는게 중복을 줄이고 더 좋을 것 같아요.

```suggestion
@media (max-width: 1120px) {
    header{
        padding: 9.5px 20px;
    }
}

@media (max-width: 744px) {
 /* 위의 조건을 상속받으므로 불필요한 중복 제거 가능 */
```

또한 min을 375px로 제한하시면 그 이하로 화면너비가 줄어들었을 때 PC 디자인이 적용되므로 min 조건을 빼시거나 html, body 태그에 min-width: 375px를 추가하시는 것을 추천드려요.